app/uk/gov/hmrc/agentclientrelationships/connectors/UsersGroupsSearchConnector.scala

```diff
@@ -51,4 +57,18 @@ class UsersGroupsSearchConnector @Inject()(
       case _: NotFoundException => Future failed RelationshipNotFound("UNKNOWN_AGENT_CODE")
     }
   }
+
+  def isAdmin(userId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] = {
+    val url = new URL(baseUrl, s"users-groups-search/users/$userId")
+    monitor("ConsumedAPI-UGS-getUserInfo-GET") {
+      httpGet.GET[CredentialRole](url.toString).map(_.credentialRole == "Admin")
+    } recoverWith {
+      case _: NotFoundException => Future failed UserNotFound("UNKNOWN_USER_ID")
```

Comment:
if the user is not found , do we really care,  as anyway that user is not going to be an Admin, right ?
so not sure if returning false here makes more sense instead of throwing an error ? :)

Reply:
yeah i see what you mean

Reply:
will fix now

Reply:
fixed